### PR TITLE
Fix typo in autogen.sh

### PR DESCRIPTION
**Pull request by kamoii**
The rust relating code in autogen.sh refers to variable `remacs_version` which is not defined, causing output to look like:
```
$ ./autogen.sh                                                              
Checking Rust toolchain install ...
error: no default toolchain configured
error: no default toolchain configured
error: no default toolchain configured
Remacs currently requires Rust toolchain version .
Run 'rustup install '.
```

### CODE REVIEW

**Comment by brotzeit** [comment]
Thanks!

File: autogen.sh [hunk 1]
```diff
@@ -190,15 +190,15 @@ if [ -n $rust_toolchain_vers_path ] ; then
 
     case $retval in
         0) echo "Your system has the required Rust toolchain installed for building Remacs." ;;
```
Comment (brotzeit) [inline autogen.sh:192]: Another remacs =)

**Comment by brotzeit** [comment]
Sorry, but I think we should use "emacs-ng" as name.

**Review by DavidDeSimone — APPROVED** [review]
LGTM

**Review by black7375 — APPROVED** [review]

**Pull request merged**